From 7ccf601c7bab3d1363f73e1a688f36ee291b8473 Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jslaby@suse.cz>
Date: Fri, 18 Dec 2020 09:57:54 +0100
Subject: [PATCH] lift "requests" version requirement
Patch-upstream: it will get fixed eventually

We have 2.25 in TW.

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 358e6a7a..f6ed6745 100644
--- a/setup.py
+++ b/setup.py
@@ -41,7 +41,7 @@ setup(
     data_files = [('share/man/man5', ['man/b4.5'])],
     keywords=['git', 'lore.kernel.org', 'patches'],
     install_requires=[
-        'requests~=2.24.0',
+        'requests>=2.24.0',
         'dkimpy~=1.0.5',
         'dnspython~=2.0.0',
     ],
-- 
2.26.2

